modules/metrics/src/main/java/com/spotify/apollo/metrics/semantic/Metric.java

```diff
@@ -19,18 +19,26 @@
  */
 package com.spotify.apollo.metrics.semantic;
 
-import com.codahale.metrics.Timer;
-import com.spotify.apollo.metrics.TimerContext;
+/**
+ * Enumerates the metrics tracked, keyed by the 'what' tag.
+ */
+public enum Metric {
```

Comment:
I think another name would be better as Metric represent something else. Maybe WhatTag?